src/network/networkprotocol.h

```diff
@@ -229,6 +229,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		[bump for 5.9.1]
 	PROTOCOL VERSION 46:
 		Move default hotbar from client-side C++ to server-side builtin Lua
+		Move death screen to server and make it a regular formspec
```

Comment:
what i see in this PR is that a new client cannot go to a old server because it won't send respawn command, but a new server can send to old client the deathscreen. Is this what we expect ? shouldn't we just say that 5.10+ only and sorry for the 5.9 and lesser compat, finishing to cleanup all this area ?

Reply:
Have you tested this PR? In my testing, the death screen worked properly, both with new server + old client and with old server + new client. 

We shouldn't introduce an avoidable compatibility breakage here.